### PR TITLE
NODE-5822 Release 5.0.3

### PR DESCRIPTION
**Pull request by alexorekhov**
Release 5.0.3

### CODE REVIEW

File: TAG [hunk 1]
```diff
@@ -1 +1 @@
-1.3.10
+1.3.11
```
Comment (braek-neck) [inline TAG:1]: Do we have changes related to the controller code?

File: helm/Chart.yaml [hunk 2]
```diff
@@ -21,8 +21,8 @@ kubeVersion: ">=1.19.16-0"
 annotations:
   artifacthub.io/images: |
     - name: sidecar
-      image: wallarm/sidecar:5.0.2-1
+      image: wallarm/sidecar:5.0.3-1
     - name: sidecar-controller
       image: wallarm/sidecar-controller:1.3.9
```
Comment (braek-neck) [inline helm/Chart.yaml:26]: we also need to bump the controller image tag

Comment (braek-neck) [reply helm/Chart.yaml:26]: https://github.com/wallarm/sidecar/pull/190/files#diff-81801117ec01136c8a49bfcd42af8e104f4efad1f2daccda9da9d960eaad5279R767
we need to bump the controller image tag the same

**Review by braek-neck — APPROVED** [review]

**Comment by xDmitriev** [comment]
@alexorekhov please, make sure that all tests passed before the merge

**Pull request merged**